### DISCUSSION TITLE
如何更新 slidev version？

### DISCUSSION THREAD

**Discussion by PassionZale** [slidevjs/slidev#1945] [Q&A] [answered]
我本地有一个 slidev 工程，devDeps 大概是这样

```json
  "devDependencies": {
    "@antfu/eslint-config": "^2.0.0",
    "@iconify/json": "^2.2.144",
    "@slidev/cli": "^0.44.0",
    "@slidev/theme-default": "^0.21.2",
    "@slidev/theme-seriph": "^0.21.3",
    "@types/markdown-it": "^13.0.1",
    "@types/node": "^20.9.3",
    "@types/prompts": "^2.4.9",
    "eslint": "^8.54.0",
    "esno": "^4.0.0",
    "execa": "^8.0.1",
    "fast-glob": "^3.3.2",
    "playwright-chromium": "^1.40.0",
    "postcss": "^8.2.14",
    "prompts": "^2.4.2",
    "rimraf": "^5.0.5",
    "typescript": "^5.3.2"
  }
```

落后了几个 `minor version`，如果我想升级到 latest version，我只需要升级 `@/slidev/*` 的三个包吗？

**Comment by kermanx** [comment]
理论上是的，但是我不能确保其他包会不会有和 Slidev 最新版不兼容。可以考虑使用 `pnpx taze --major --write` 更新所有包到最新版

**Reply by PassionZale** [reply]
emmm... 可以帮我看下吗，仓库在这里 [PassionZale/talks](https://github.com/PassionZale/talks)。

我之前是拿 [antfu/talks](https://github.com/antfu/talks) 改了下做的这个 repo，时间有些久了，我是不是可以偷懒直接复制他的 [pnpm-workspace.yaml](https://github.com/antfu/talks/blob/main/pnpm-workspace.yaml) 🫣

升级主要是想要用上 [Click Markers](https://sli.dev/features/click-marker)

**Reply by kermanx** [reply, marked as answer]
运行
```sh
pnpx taze --major --write
pnpm i
```

一般没问题。用他的 pnpm-workspace.yaml 的话，package.json 也要相应改。你可以自己试试

**Reply by PassionZale** [reply]
好的 感谢~
